<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PAM123" name="Default test" tests="3" failures="1" timestamp="11 May 2017 12:52:08 GMT" time="129.042" errors="0">
  <testcase name="placeOrderBySearch" time="124.086" classname="app.fynd.Tests.PlaceOrderSanity">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 6, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 6, Size: 1
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at io.appium.java_client.pagefactory.ElementListInterceptor.getObject(ElementListInterceptor.java:39)
	at io.appium.java_client.pagefactory.interceptors.InterceptorOfAListOfElements.intercept(InterceptorOfAListOfElements.java:51)
	at $java.util.ArrayList$$EnhancerByCGLIB$$c0a2d785.get(<generated>)
	at app.fynd.Pages.MajorFlowSanityPage.selectPaymentMethod(MajorFlowSanityPage.java:660)
	at app.fynd.Tests.PlaceOrderSanity.placeOrderBySearch(PlaceOrderSanity.java:87)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- placeOrderBySearch -->
  <testcase name="placeOrderByBanner" time="0.001" classname="app.fynd.Tests.PlaceOrderSanity">
    <skipped/>
  </testcase> <!-- placeOrderByBanner -->
  <testcase name="placeOrderByNotification" time="0.001" classname="app.fynd.Tests.PlaceOrderSanity">
    <skipped/>
  </testcase> <!-- placeOrderByNotification -->
</testsuite> <!-- Default test -->
